### DISCUSSION TITLE
Is it possible to disable Dark mode for .cbz files?

### DISCUSSION THREAD

**Discussion by vlkvch** [koreader/koreader#9334] [Q&A] [answered]
Can I have Dark mode enabled for every file extension, but `.cbz`?

**Comment by poire-z** [comment, marked as answer]
Not with current KOReader.
But you can get that with your KOReader and a user patch (see https://github.com/koreader/koreader/issues?q=label%3A%22User+patch+available%22 for how-to):
`koreader/patches/2-no-nightmode-with-cbz.lua`

```lua
local Event = require("ui/event")
local ReaderView = require("apps/reader/modules/readerview")
local util = require("util")

local onReaderReady_orig = ReaderView.onReaderReady
ReaderView.onReaderReady = function(self)
    onReaderReady_orig(self)
    local ext = util.getFileNameSuffix(self.ui.document.file)
    local nightmode_on = true
    if ext == "cbz" then
        nightmode_on = false
    end
    self.ui:handleEvent(Event:new("SetNightMode", nightmode_on))
end
```

**Reply by vlkvch** [reply]
As I understand, I need to switch to the Developer Upgrade mode. If it is true, how can I do it?

**Reply by poire-z** [reply]
No, I don't think you need to change anything.
Just create the koreader/patches/ directory and a file in it with the above name and content, and see if it works.

**Reply by vlkvch** [reply]
I followed the instructions, but the patch does not seem to work.

**Reply by poire-z** [reply]
Oh right, it was not in stable v2022.06 - so yes, you need to get a more recent KOReader.
I think upgrading/switching to dev channel can be done via some menu in the right most top menu panel.
